### PR TITLE
Update fastcheck and linters configurations

### PR DESCRIPTION
**Pull request by coorasse**

### CODE REVIEW

File: ruby_on_rails/linting_and_automatic_check.md [hunk 1]
```diff
@@ -32,14 +32,16 @@ bundle exec rubocop -D
 
 ## Brakeman
 
-```ruby
-group :development, :test do
-  gem 'brakeman', require: false
-end
+Brakeman comes byb default with Rails. Add it to the `bin/fastcheck` script.
```
Comment (CuddlyBunion341) [inline ruby_on_rails/linting_and_automatic_check.md:35]:
```suggestion
Brakeman comes by default with Rails. Add it to the `bin/fastcheck` script.
```

File: ruby_on_rails/linting_and_automatic_check.md [hunk 2]
```diff
@@ -32,14 +32,16 @@ bundle exec rubocop -D
...
+
+```
+bundle exec brakeman -q -z --no-summary --no-pager
 ```
 
 ## Mdl
 
+An optional check for markdown files. You can include it or not. Discuss within your team.
```
Comment (CuddlyBunion341) [inline ruby_on_rails/linting_and_automatic_check.md:43]: Maybe add example usage of mdl?
```
bundle exec mdl $(find . -name '*.md'|grep -E -v '(SUMMARY.md|vendor/)')
```

File: ruby_on_rails/linting_and_automatic_check.md [hunk 3]
```diff
@@ -48,14 +50,12 @@ end
 
 ## SCSS lint
 
-To lint the SASS/SCSS files in our project we are going to use the `sass-lint` npm package.
-
-`bin/yarn add sass-lint`
+To lint the SASS/SCSS files in our project we are going to use the `stylelint` npm package.
```
Comment (CuddlyBunion341) [inline ruby_on_rails/linting_and_automatic_check.md:53]:
```suggestion
To lint the SASS/SCSS files in our project you can use the `stylelint` npm package.
```

File: templates/bin/fastcheck [hunk 4]
```diff
@@ -13,11 +13,8 @@ fi
 bundle exec brakeman -q -z --no-summary --no-pager
 bundle exec mdl ./ -g
 
-# old SCSS linter (deprecated)
-bundle exec scss-lint `find app/assets/stylesheets/ -name "*.scss"`
-
-# new SASS linter
-yarn sass-lint -v --no-exit app/javascripts/**/*.scss
+# for scss
+npx stylelint "./app/assets/stylesheets/**/*.scss"
```
Comment (CuddlyBunion341) [inline templates/bin/fastcheck:17]: Previously the dependency was added using the yarn binstub, why use npm here?
```suggestion
yarn run stylelint "./app/assets/stylesheets/**/*.scss"
```

**Review by CuddlyBunion341 — APPROVED** [review]
Looks good to me 👍🏻!

File: ruby_on_rails/linting_and_automatic_check.md (hunk 2 above)
Comment (coorasse) [reply ruby_on_rails/linting_and_automatic_check.md:43]: The template includes an example. I don't want to duplicate it here

**Pull request merged**